.github/workflows/env-checker-ci-schedule.yml

```diff
@@ -21,8 +21,9 @@ jobs:
       matrix: # TODO: add more versions and cases where Node.js do not exist
         os: [windows-latest, macos-latest, macos-11, ubuntu-latest]
         node-version: [10, 12, 14]
-        func-version: [none]
+        func-version: [none, 2, 3]
```

Comment:
I remember we do not support func v2, right?

Reply:
right, but we need to test  **old version**